Identity_Presentation/Controllers/AuthenticationController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Identity_Application.Commands.Authentication;
+using Identity_Application.Models.AuthorizationModels;
+using Identity_Application.Queries.Authentication;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Identity_Presentation.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class AuthenticationController : Controller
+{
+    private readonly IMediator _mediator;
+
+    public AuthenticationController(IMediator mediator)
+    {
+        _mediator = mediator;
+    }
+
+    [AllowAnonymous]
+    [HttpPost("register")]
+    public async Task<IActionResult> Register([FromBody] RegisterVM vm)
```

Comment:
in future can consider way how to prevent attack on AllowAnonymous actions

Reply:
We can add rate limits and CORS policy details to defend the endpoints from the attacks